policyengine_us/parameters/gov/states/vt/tax/income/rates/alternative_minimum_tax/income_threshold.yaml

```diff
@@ -0,0 +1,14 @@
+description: Vermont qualifies filers for the alternative minimum tax with adjusted gross income above this threshold.
+metadata:
+  label: Vermont alternative minimum tax income threshold
+  period: year
+  unit: currency-USD
+  reference:
+    - title: Vermont §5822. Tax on income of individuals, estates, and trusts (a)(6)
+      href: https://legislature.vermont.gov/statutes/section/32/151/05822
```

Comment:
can you double check the link, I can't open it

Reply:
![image](https://github.com/PolicyEngine/policyengine-us/assets/66671734/b6b85bc2-b6b0-45b5-a95a-c3d8662d20ce)
I can actually open the link. Maybe because the server is closed at night. You want me to change another source? I can do so in another commit.